src/commands/ban.rs

```diff
@@ -38,10 +38,10 @@ async fn do_ban(
         .invalid_usage(&BAN_COMMAND_OPTIONS)?;
 
     let permission_level = checks::get_permission_level(&ctx, &msg).await?;
-    if permission_level == PermissionLevel::Helper {
-        if Utc::now().signed_duration_since(mentioned_user.created_at()) > Duration::days(3) {
-            abort_with!("You can't ban an account older than 3 days");
-        }
+    if permission_level == PermissionLevel::Helper
+        && Utc::now().signed_duration_since(mentioned_user.created_at()) > Duration::days(3)
+    {
+        abort_with!("You can't ban an account older than 3 days");
     }
```

Comment:
This is one of the cases where i strongly disagree with clippy,... here it's just cleaner to have them as nested if, because those two checks are fully unrelated and the outer check is an idiom that is used throughout the code. consistency matters,...

but it's probably not worth a `#[allow(clippy::whatever_this_rule_is_called)]` so fuck it 🤷🏻‍♂️